apps/mark/backend/src/env.d.ts

```diff
@@ -3,7 +3,6 @@ declare namespace NodeJS {
     readonly CI?: string;
     readonly NODE_ENV: 'development' | 'production' | 'test';
     readonly PORT?: string;
-    readonly VX_APP_MODE?: string;
```

Comment:
Can also remove from vxsuite-complete-system once this is merged

Reply:
Thanks for the reminder!